transactions/views.py

```diff
@@ -395,11 +399,16 @@ def foreign_card_validation_response(request):
         references = {k: v for k, v in data.items() if k.endswith('ref')}
         amount = data.get('amount', 0)
         save_deposit(purpose, references, amount, user, property)
+
     else:
-        return Response({'message': verify_resp.get('message')},
-                        status=status.HTTP_400_BAD_REQUEST)
+        return HttpResponseRedirect(
+            f"{domain}/international-payment/status/" +
+            f"?message={verify_resp.get('message')}&status=fail"
```

Comment:
Hey @patrickf949, could we make this status as failure instead of fail?, It sounds grammatically incorrect.

Reply:
@CryceTruly done